src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -17075,6 +17108,18 @@ public void run3() {
                 ItemList.Circuit_Parts_DiodeXSMD.get(64L),
                 160,
                 120000);
+        // Inductor
+        GT_Values.RA.addAssemblerRecipe(
+                new ItemStack[] {
+                    GT_ModHandler.getModItem("miscutils", "itemFoilTitansteel", 4L),
+                    GT_ModHandler.getModItem("miscutils", "itemFoilBotmium", 1L),
```

Comment:
botmium and titansteel are the uv tier alloy while all other xsmd material is zpm/luv tier. imo it is gated too later. i suggested to use artherium-sn and hikarium foil which is a zpm/luv material

Reply:
Consistency with the other xsmd components is the most important thing. I wasn't sure what to use as I've never played with any of these materials/components. If you think those are better, I'm all for it.